sentry/src/main/java/io/sentry/logger/LoggerApi.java

```diff
@@ -32,8 +32,8 @@ public LoggerApi(final @NotNull Scopes scopes) {
 
   @Override
   public void trace(final @Nullable String message, final @Nullable Object... args) {
-    // TODO SentryLevel.TRACE does not exists yet
-    //    log(SentryLevel.TRACE, message, args);
+    // TODO SentryLevel.TRACE does not exists yet so we just report it as DEBUG for now
+    log(SentryLevel.DEBUG, message, args);
```

Comment:
I opted for this as a quick fix to get `trace()` working. We can have a proper fix later but we'll have to check what consequences this has for the rest of the SDK since the enum is used in multiple places already.

Reply:
Updated in https://github.com/getsentry/sentry-java/pull/4392